src/tasks/components/PageBreadcrumbs.tsx

```diff
@@ -0,0 +1,41 @@
+import React, {FC} from 'react'
+
+// Components
+import {FlexBox, FlexDirection} from '@influxdata/clockface'
+
+type PageInfo = {
+  text: string
+  onClick?: () => void
+}
+
+type Props = {
+  pages: Array<PageInfo>
+}
+
+const PageBreadcrumbs: FC<Props> = ({pages}) => {
+  const getCrumb = (page: PageInfo, isItTheLastPage: boolean) => {
+    const text = `${page.text}${!isItTheLastPage ? ' >' : ''}`
```

Comment:
Thanks Bucky! 
Two routes we can go :
1. Leave this in and replace it with a clockface component later
2. fix this now. 

I am okay with either options for this. 

How we might fix it: 
- To prevent screen reader announcement of the visual separators between links, I believe we can add them via CSS.  I,E `:after { content: " > " }`

Reply:
Also wrapping the breadcrumb in a <nav> with proper aria label `aria-label="Breadcrumb"` with each page being a <a>  and marking the current page with `aria-current="page"` should help in the department

Reply:
helpful source : https://www.w3.org/TR/wai-aria-practices-1.1/examples/breadcrumb/index.html

Reply:
thank you @daviddkkim, Mr. Accessibility!